public/src/components/calls/pre_call_btn_grp.jsx

```diff
@@ -46,38 +46,61 @@ export default class PreCallButtonGroup extends Component {
             releaseCall,
             status,
             endVolunterTwilioCon,
-            clearCurrentCall } = this.props;
-    clearCurrentCall();
-    releaseCall(user_id, campaign_id, call_id, status);
+            clearCurrentCall,
+            enableCallControl } = this.props;
+    if (status === 'ACTIVE') {
```

Comment:
I'm not sure ACTIVE is a valid status. Did you mean IN_PROGRESS? Or am I completely off kilter?

Reply:
actually I think I meant `current_call`.

changing.

Reply:
Or did you mean to release a non-ATTEMPTED call. Not sure what the intended effect is here.